src/EnergyPlus/Autosizing/SystemAirFlowSizing.cc

```diff
@@ -797,21 +795,20 @@ Real64 SystemAirFlowSizer::size(EnergyPlusData &state, Real64 _originalValue, bo
                     }
 
                 } else {
+                    auto &sysSizPeakDDNum = state.dataSize->SysSizPeakDDNum(this->curSysNum);
```

Comment:
This is correct. Although the fact that the object iself is called `Num` suggesting it is a number rather than an object is confusing.